packages/router/src/bindings/contractReader/index.ts

```diff
@@ -214,6 +214,20 @@ export const handleSingle = async (
     const preparePayload: PreparePayload = _transaction.payload;
     try {
       logger.info("Preparing receiver", requestContext, methodContext);
+      const didConfirm = cache.auctions.confirmBid(
```

Comment:
This seems a bit unnecessary? Our signature on a bid is essentially our commitment, so I'm not sure what we gain by confirming that it's in the cache.

Reply:
You're misreading the purpose of this. It's not for our sake here, it's for the sake of keeping that outstanding liquidity "held" as outstanding until we've confirmed that our prepare has deducted from our overall balance.

Reply:
https://github.com/connext/nxtp/pull/700/files/7f1b5df049f918c48c9a8f326788ae93b955dc90#r778491800

Reply:
So:
`bid sent ---> bid accepted ---> user prepares ---> router sees user prepared -***-> router prepares and liquidity is deducted from router's pool`
***at some point here, asynchronously, the bid could "expire" but we want the liquidity to remain outstanding until our prepare is complete

Reply:
I'll add a clarifying comment